app/controllers/api/base_controller.rb

```diff
@@ -54,5 +53,50 @@ def set_access_control_headers
     def collection_config
       @collection_config ||= CollectionConfig.new
     end
+
+    def log_request_initiated
```

Comment:
These could have probably stayed in the logger concern to at least keep them bucketed together.